services/shareService.ts

```
@@ -0,0 +1,101 @@
+				);
+				throw new Error(`ルームデータの取得に失敗しました: ${error.message}`);
+			}
+			console.error("予期せぬエラーが発生しました:", error);
+			throw new Error(`予期せぬエラーが発生しました`);
+		}
+	},
+);
+
+export const createShareRoom = createAsyncThunk(
+	"share/createShareRoom",
+	async ({ userId, sharedRoomTitle }: CreateShareRoomRequest) => {
```

Biome (1.9.4)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)